cmd/main.go

```diff
@@ -77,6 +84,8 @@ func main() {
 		"If set the metrics endpoint is served securely")
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
+	flag.StringVar(&kubeconfigSecretLabel, "kubeconfig-secret-label", "sigs.k8s.io/multicluster-runtime-kubeconfig", "The label of the kubeconfig secret.")
+	flag.StringVar(&kubeconfigSecretKey, "kubeconfig-secret-key", "kubeconfig", "The key of the kubeconfig secret.")
```

Comment:
These should go into the config file not a cli flag

Reply:
Or just hard code an opinionated value?